packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -275,18 +255,23 @@ class ChartController extends DisposableController
 
   String title;
 
-  displayDuration zoomDuration = displayDuration.live;
+  /// zoomDuration values of:
+  ///     null implies all (default)
+  ///     Duration() imples live
+  ///     Duration(minutes: 5) implies 5 minute interval
+  ///     Duration(minutes: 15) implies 15 minute interval
+  Duration _zoomDuration = const Duration();
+
+  Duration get zoomDuration => _zoomDuration;
 
   static const oneMinuteInMs = 1000 * 60;
 
-  void setXAxisZoomDuration(Duration duration) {
+  set zoomDuration(Duration duration) {
```

Comment:
seems surprising that this method can only be called with certain very specific duration values. Comment indicating what the values are or make it more robust.

Reply:
Changed to accept any duration.